mm2src/mm2_main/src/lp_swap/taker_swap.rs

```diff
@@ -1974,41 +2006,45 @@ impl TakerSwap {
                 ),
             },
             None => {
-                let can_refund = try_s!(self.taker_coin.can_refund_htlc(taker_payment_lock).compat().await);
-                if let CanRefundHtlc::HaveToWait(seconds_to_wait) = can_refund {
-                    return ERR!("Too early to refund, wait until {}", now_ms() / 1000 + seconds_to_wait);
-                }
+                if self.taker_coin.is_auto_refundable() {
+                    ERR!("Taker payment will be refunded automatically!")
+                } else {
```

Comment:
Could you please refactor this as
```rust
if self.taker_coin.is_auto_refundable() {
    return ERR!("Taker payment will be refunded automatically!");
}

let can_refund = try_s!(self.taker_coin.can_refund_htlc(taker_payment_lock).compat().await);
```
This will allow to avoid whitespace changes :)